examples/models/image_classification/CNN.py

```diff
@@ -0,0 +1,174 @@
+import base64
+import numpy as np
+import tensorflow as tf
+from tensorflow import keras
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import confusion_matrix
+import itertools
+from keras.layers import Dense, Dropout, Flatten, Conv2D, MaxPool2D
+from keras import layers
+from keras.layers.normalization import BatchNormalization
+from keras import models
+from keras.utils.np_utils import to_categorical
+from keras.optimizers import RMSprop
+from keras.preprocessing.image import ImageDataGenerator
+from keras.callbacks import ReduceLROnPlateau
+
+from rafiki.model import BaseModel, InvalidModelParamsException, test_model_class, \
+                        CategoricalKnob, FixedKnob, dataset_utils, logger
+from rafiki.constants import TaskType, ModelDependency
+
+class CNN(BaseModel):
+    '''
+    Implements CNN using keras for simple image classification
+    '''
+    @staticmethod
+    def get_knob_config():
+        return {
+            'epochs': FixedKnob(100),
+            'batch_size': FixedKnob(86),
```

Comment:
Given that one of the strengths of Rafiki automating the hyperparameter tuning process for models, and fine-tuning it for application developers' datasets, I think we should add more knobs here and set appropriate value ranges e.g. `batch_size` can be `IntegerKnob`.